packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -3447,6 +3459,23 @@ export default class Meeting extends StatelessWebexPlugin {
     return false;
   }
 
+  /**
+ * Check if the meeting supports the Reactions
+ * @returns {boolean}
+ * @throws ReactionsNotSupportedError
```

Comment:
I don't see it throwing anything? Maybe remove this comment

Reply:
Looks like `isTranscriptionSupported` incorrect as well.

`@throws` needed only when the function throws error with `throw` keyword. Here the error handled with logging and returning `false`, no throwing